harris/harris.go

```diff
@@ -0,0 +1,195 @@
+package harris
+
+import (
+	"yi/matrix"
+	"image"
+	"yi/point"
+	"log"
+)
+const k = 0.04
+
+func smoothFilter() matrix.Matrix {
+	result := matrix.Ones(3, 3).Scale(float64(1)/float64(9))
```

Comment:
No need to explicitly cast like this; untyped constants are dynamic enough to pick the correct type when used

https://play.golang.org/p/L0slV5DMc_